example/apu2_gpiod.c

```diff
@@ -0,0 +1,236 @@
+/* **********************************************************************
+**
+** Copyright (c) 2021 Raimo Niskanen <raimo (at) niskanen (dot) nu>
+**
+** Permission to use, copy, modify, and distribute this software for any
+** purpose with or without fee is hereby granted, provided that the above
+** copyright notice and this permission notice appear in all copies.
+**
+** THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+** WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+** MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+** ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+** WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+** ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+** OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+**
+** **********************************************************************
+**
+** apu2_gpiod - a daemon to control the APU2/3 LEDs on OpenBSD
+**
+** WARNING: Do not use this code on any other hardware than
+** PC Engines APU2 (tested) or APU3 (should work).
+** The direct memory accesses may do anything including damage
+** to other hardware.
+**
+** See apu_gpio_lib from PC Engines.  This program is built
+** as one of its examples, and can be copied to for example
+** /root/bin/.
+**
+** Start the daemon from rc.securelevel while direct memory
+** access is allowed.
+**
+** Example rc.securelevel line:
+**     /root/bin/apu2_gpiod /var/apu2_gpio
```

Comment:
I had to create a file /etc/rc.securelevel

Reply:
Yes.  It does not exist per default.
Although is documented in rc(8) with an alias rc.securelevel, the man page does not mention the detail that it does not exist per default.

Reply:
Improved in [1ef622d](https://github.com/pcengines/apu_gpio_lib/pull/4/commits/1ef622df51e993b9c2d5f892136f2951c18d3e45)

Reply:
@Pokisiekk Please confirm, so we can close it.

Reply:
I confirm